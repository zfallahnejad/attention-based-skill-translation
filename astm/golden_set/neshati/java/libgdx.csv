960524,libgdx
1816356,libgdx
1895303,libgdx
506796,libgdx
3182091,libgdx
2327745,libgdx
2512687,libgdx
1804251,libgdx
4140572,libgdx
2399024,libgdx
816542,libgdx
416549,libgdx
1676123,libgdx
1800173,libgdx
2394933,libgdx
