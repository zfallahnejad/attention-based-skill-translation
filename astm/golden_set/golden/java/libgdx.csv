960524,105,79,0.5706521739130435
1816356,98,32,0.7538461538461538
1895303,62,70,0.4696969696969697
506796,53,33,0.6162790697674418
3182091,40,46,0.46511627906976744
2327745,32,30,0.5161290322580645
2512687,32,3,0.9142857142857143
1804251,28,35,0.4444444444444444
4140572,25,36,0.4098360655737705
2399024,18,9,0.6666666666666666
816542,17,8,0.68
416549,14,5,0.7368421052631579
1676123,11,2,0.8461538461538461
2394933,11,8,0.5789473684210527
1800173,11,5,0.6875
531398,10,3,0.7692307692307693
1906584,10,12,0.45454545454545453
729460,10,0,1.0
